pages/help.js

```diff
@@ -68,7 +69,7 @@ export default function Help() {
           </li>
         </div>
         <h1 className="text-2xl text-white">
-          <u>After you study:</u>
+          <u>After You Study:</u>
```

Comment:
Doesn't need to be capitalized.

Reply:
To look good with the rest of the page, it should be.

Reply:
I don't actually think this is the case... we have other non caps text.

Reply:
Not if you scroll up a bit

Reply:
It just looks weird on the rest of the page with all the other capilizations